kuka_sunrise_fri_driver/test/test_driver_activation.py

```diff
@@ -71,7 +71,6 @@ def generate_test_description():
 class TestDriverActivation(unittest.TestCase):
     def test_read_stdout(self, proc_output):
         # Check for successful initialization
-        proc_output.assertWaitFor("got segment base", timeout=5)
```

Comment:
the new log messages in jazzy are e.g.
`[robot_state_publisher-3] [INFO] [1721996934.659519468] [robot_state_publisher]: Robot initialized`
please fix tests to wait for this message

Reply:
Done.
Note: I checked the kroshu upstream repos as well, the robot descriptions also use the "got segment" check for ros-in frames:
```
class TestModels(unittest.TestCase):
    def test_read_stdout(self, proc_output):
        # Check for frames defined by ROS-Industrial
        proc_output.assertWaitFor("got segment base", timeout=5)
        proc_output.assertWaitFor("got segment flange", timeout=5)
        proc_output.assertWaitFor("got segment tool0", timeout=5)
``` 

This might need some additional changes to confirm the given frames.

Also, concerning the multi robot checks, let me know if you need further adjustments as it is currently impossible to distinguish between the two robots for the "Robot initialized" test (all the other test cases check both robots as in those cases the 2 robots post different outputs)

Reply:
Is it possible to ensure that the message arrived twice? that would be enough, if not, we should come back to this later

Reply:
I checked, it is possible to supply additional parameters, so by defining the process to check it is possible to distinguish between the 2 robots. The process names seem to be consistent both locally and on the ros-in-ci. However, I don't think that on the long run it would be an effective way of testing. eg:
```
proc_output.assertWaitFor("Robot initialized", process="robot_state_publisher-3", timeout=20)
proc_output.assertWaitFor("Robot initialized", process="robot_state_publisher-12", timeout=20)
```
So, I think checking for the same message twice is the way to go, even though it leaves something to be desired.

Reply:
So basically the assertWaitFor also checks previous logs, which means that calling it twice doesn't matter, the second call will also see the first "Robot initialized" message and pass